.github/workflows/testing.yml

```
@@ -62,12 +62,12 @@ jobs:
           set -vxeo pipefail
           conda env list
           mamba install -c conda-forge -y numpy matplotlib
-          pip install -v pypylon
+          pip install -r requirements.txt
+          pip install -r requirements-dev.txt
           pip list
           conda list
 
       - name: Test the code
         run: |
           set -vxuo pipefail
-          wget ${{ secrets.PYLON_GRAB_SCRIPT }}/pylon_grab.py -O pylon_grab.py
-          python pylon_grab.py
+          pytest
```

Let's make it a bit more verbose so that it can print the output too.
```suggestion
          pytest -vv -s
```